tests/framework/pvc.go

```diff
@@ -42,6 +43,18 @@ func (f *Framework) CreateBoundPVCFromDefinition(def *k8sv1.PersistentVolumeClai
 	return pvc
 }
 
+// CreateBoundPopulationPVCFromDefinition is a wrapper around utils.CreatePVCFromDefinition that also force binds pvc on
+// on WaitForFirstConsumer storage class by executing f.ForceBindIfWaitForFirstConsumer(pvc)
+func (f *Framework) CreateBoundPopulationPVCFromDefinition(def *k8sv1.PersistentVolumeClaim) *k8sv1.PersistentVolumeClaim {
```

Comment:
Just making sure I understand:
With populators, Bound means the population is done, so there is no way the original target PVC
will ever be bound before somebody `curl`d the upload endpoint and that process completed successfully?

This was not a problem with Immediate binding storage classes because the whole `ForceBindIfWaitForFirstConsumer` logic was skipped?

Reply:
exactly, also the name of this function is probably not accurate, ill change that.
But yet with populators the PVC is bound only after the population, hence you cant wait here for the consumer pod to be running, I create here the consumer pod just to get the selectedNode annotation to be able to do the population